src/formatting/formatter/datetime.rs

```diff
@@ -108,20 +152,32 @@ impl Formatter for DatetimeFormatter {
             datetime: DateTime<T>,
         ) -> Result<String, FormatError>
         where
-            T: TimeZone,
+            T: TimeZone + TimezoneName<T>,
```

Comment:
Why is `TimezoneName` generic over `T`, if `T` must be `Self`?

Reply:
just remnants from a previous test, I'll remove them.